extensions/amp-story/1.0/amp-story.js

```diff
@@ -297,6 +297,9 @@ export class AmpStory extends AMP.BaseElement {
 
     /** @private {?UIType} */
     this.uiState_ = null;
+
+    /** @private {boolean} whether the styles were rewritten */
+    this.rewroteStyles_ = false;
```

Comment:
super nitty but it always get harder to read/write conjugated verbs, how about `didRewriteStyles_` that's a more widespread pattern?